### ISSUE TITLE
Left menu does not expand sections

### ISSUE THREAD

**Issue by psss** [readthedocs/sphinx_rtd_theme#1525]
### Problem

After updating to `sphinx-rtd-theme==1.3.0` I've noticed the page sections are no more expanded in the left menu:

 * old: https://did.readthedocs.io/en/stable/overview.html#format
 * new: https://did--321.org.readthedocs.build/en/321/overview.html#format

### Reproducible Project

Please give a link to a public reproducible project or provide an example of the Restructured Text that gives an issue.

https://github.com/psss/did/blob/main/README.rst?plain=1#L82

#### Error Logs/Results

Before:
![image](https://github.com/readthedocs/sphinx_rtd_theme/assets/2803150/24ff573f-1fdb-452e-aa24-23add6da49eb)

After:
![image](https://github.com/readthedocs/sphinx_rtd_theme/assets/2803150/f7ddb3fb-91d8-4bd4-9332-ec938f540ee9)

#### Expected Results

Be able to expand sections in the left menu, see highlighted last
clicked section.

### Environment Info

- Python Version: 3.11
- Sphinx Version: 7.2.4
- RTD Theme Version: 1.3.0

**Comment by jdillard** [comment]
Did you also upgrade Sphinx? In Sphinx 6 jQuery was removed and you have this error:
```
Uncaught ReferenceError: jQuery is not defined
```
Make sure to read changelogs when upgrading.

**Comment by psss** [comment]
Yes, I've upgraded `sphinx` to `7.2.4`. So, this means that the menu expansion is not expected to work with the latest packages?

**Comment by brenj** [comment]
I recently updated this package and sphinx to the latest and had this issue as well. What fixed it for me was adding `sphinx_rtd_theme` as an extension. It's mentioned here: https://sphinx-rtd-theme.readthedocs.io/en/stable/installing.html. Maybe the README should be updated to match this if that is required. Hope that helps.

**Comment by psss** [comment]
> What fixed it for me was adding `sphinx_rtd_theme` as an extension.

@brenj, thank you very much for the hint. It helped and now it works again!
